payments/swish/after-payment.md

```diff
@@ -20,20 +20,19 @@ sidebar:
 
 ## Payment Resource
 
-The `payment` resource and all general sub-resources can be found in the
-[core payment resources][core-payment-resources] section.
+This section describes the general sub-resources of the API that are used to
+generate payment requests.
 
-### Create Payment
+## Create Payment
```

Comment:
This is before "after payment" ? move this out ? maybe to other features or lets divide this into redirect/direct

Reply:
It's been moved